runsc/sandbox/sandbox.go

```diff
@@ -406,6 +406,9 @@ func (s *Sandbox) createSandboxProcess(conf *config.Config, args *Args, startSyn
 		cmd.Args = append(cmd.Args, "--panic-log-fd="+strconv.Itoa(nextFD))
 		nextFD++
 	}
+	if conf.Traceback != "" {
+		cmd.Args = append(cmd.Args, "--traceback", conf.Traceback)
+	}
```

Comment:
This is not needed. Flags in Config are propagated above when `Config.ToFlags()` is called.